### DISCUSSION TITLE
how to slience info level log

### DISCUSSION THREAD

**Discussion by lfxx** [mosecorg/mosec#392] [General]
When start a server,a lots of log will be printed.Any way to slience those info level logs,can you supply a level choice?

**Comment by kemingy** [comment]
It seems the current `get_logger` function doesn't provide a log-level parameter. I'll create a PR to fix it.

**Reply by lfxx** [reply]
Thanks very much.BTW,setup.py LINE67 seems to modify to 
```
shutil.copytree(build_libpath, libpath, dirs_exist_ok=True)
```
otherwise,`PRODUCTION_MODE=1 pip install -e .` will get an error as 
```
error: [Errno 21] Is a directory: '/tmp/tmpkytw3woy.build-lib/mosec/bin
```

**Reply by lfxx** [reply]
i think PRODUCTION_MODE should be set as default since if mosec is not build,server can not be started.

**Reply by kemingy** [reply]
For local development, you can refer to the [contribution guide](https://github.com/mosecorg/mosec/blob/main/CONTRIBUTING.md).

`setup.py` file is not intended to be used directly since it's complex to build the cross-compiled package. We use the cibulidwheel CI to do this.

**Comment by lfxx** [comment]
> For local development, you can refer to the [contribution guide](https://github.com/mosecorg/mosec/blob/main/CONTRIBUTING.md).
> 
> `setup.py` file is not intended to be used directly since it's complex to build the cross-compiled package. We use the cibulidwheel CI to do this.

solved.Thanks!
